### ISSUE TITLE
Project 1: Jupyter environment on Windows 7 with inconsistent results & others suggestions

### ISSUE THREAD

**Issue by pgordon245** [udacity/sdc-issue-reports#393]
Hello Udacity,
Jan first term student, so yes here comes the questions.  However, first off, want to say thanks for setting up this program.  Can see a lot of hard work and know you are doing your best to make this a success for everyone.  So, thanks!

With that, wanted to point out heading near my project 1 deadline and have had nothing but confusing and frustration.  From incorrect pointers to the wrong project, to an inconsistent test platform, it is been an absolute nightmare.  I really want to learn and be involved with this program, but if this is a sign of things to come, we need to talk. 
 
In trying to be solution more than a problem, wanted to point a few things out:

1.	On your page 18, you call for students to download the starter kit from Github and run the jupyter notebook test.ipynb.  Sounds easy enough, but this starter kit did NOT have a test.ipynb.  (This is where things began to go off track.)  I posted the question on the ND13 applied-january slack (<= my fault for not knowing there was a different one for SDC students until last night).  Someone pointed me to another starter-kit which in fact did have the test.ipynb.  So, spend the next 4 days working on trying to get the Hough transform and video section to work!  (You can also view those questions on ND13 slack as well).  It wasn’t until the Tuesday I learned I have been working on the wrong project and needed to complete a P1 project on the next page!  If you follow the pages, I do not select next until completed, this is what happens.  So, can you PLEASE double check what is needed on these pages and what you expect from the students.  Also, it seems you have many versions of the starter kit out there.  Would be best to remove any old or not needed kits.

2.	Jupyter environment on Windows 7 is painful.  Spent hours going over and over my code and wondering why any major changes to my mask were not happening.  I would clear cache, restart the kernel, even reload the notebook.  At some point, it would magically start working and other times nothing.  Understand we need to setup our own environments, but seems Windows may still need some debugging.  Also for the first project, you may want to consider students use the browser for code.

3.	Build new student confidence.  If you want to keep your students, would recommend some changes to this first project.  Again, understand trial by fire, and well placed for later sessions.  However, when there are incorrect details, unstable environments, and requirements to produce output from areas that were NOT covered in the lessons, under a deadline is a LOT for newbies.  This causes major frustration!  Simply cover use in video streams, and maybe have the first project due in the browser.  Break out the environment setup as a different lesson that includes real working code out of the box.  Hope that Udacity’s idea of a learning platform is not guessing, googling, and hoping for insight on forums.  Expected a bit more from a paid education viewpoints.  

In short, pressing on and will try to hack away at getting something completed.  Hope this feedback is helpful.  Hope future projects cover what was in lessons and not simply out of scope challenges under the gun.  If that is your format of learning, please let me know now.  Thanks

**Comment by domluna** [comment]
@pgordon245 Sorry for the trouble you've had. 

https://github.com/udacity/CarND-Term1-Starter-Kit and https://github.com/udacity/CarND-Term1-Starter-Kit-Test are two seprate repos, each serving the different purpose. The former to setup the software environment and the latter to test the packages are functional. I've added additional notes to the page which should alleviate this confusion.

> So, spend the next 4 days working on trying to get the Hough transform and video section to work!

Were you working on `test.ipynb`? I've made an additional note on the page stating this notebook isn't the project.

Windows 7 has been a little problematic at times. There was a recent PR to the starter kit to resolve some issues although I'm not sure what you're experiencing is the same. https://github.com/udacity/CarND-Term1-Starter-Kit/commit/26758577674b0d7432de91c4530dac9bf1b0580b

**Comment by pgordon245** [comment]
Thanks Dominique for the response.  Yes, was working on test.ipynb, but have since moved onto the correct project, which is P1.ipynb.  However, still dealing with  inconsistencies so it seems.

**Comment by domluna** [comment]
Please keep sharing these inconsistencies! Although, I think it may be more constructive if they're reported as separate issues (easier to keep track).
